index.css

```diff
@@ -0,0 +1,83 @@
+.landing-page {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  flex-direction: column;
+  gap: 1rem;
+  width: 100%;
+  flex-wrap: wrap;
+}
+.main-logo {
+  font-size: 7rem;
+  font-weight: bold;
+  padding: 1rem;
+}
+
+.main-logo::first-letter {
+  color: #9370db;
+  font-size: 9rem;
+}
+
+.tag-line {
+  font-size: 1.3rem;
+  padding: 2rem;
+  margin: 1rem;
+  text-align: center;
+  width: 50%;
+  line-height: 2rem;
+}
+
+.design-text {
+  color: #f0739d;
+  font-size: 9rem;
+}
+
+.motto {
+  font-size: 1.9rem;
+  padding: 1rem;
+}
+
+@media only screen and (max-width: 986px) {
+  .main-logo {
+    font-size: 4rem;
+  }
+  .main-logo::first-letter {
+    color: #9370db;
+    font-size: 5rem;
+  }
+  .design-text {
+    color: #f0739d;
+    font-size: 5rem;
+  }
```

Comment:
color is already defined for these properties, outside the media query.

Reply:
Yes. My bad. I will remove them.